packages/polkadart_scale_codec/test/functions_test/extensions_test.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:test/test.dart';
+import 'package:polkadart_scale_codec/polkadart_scale_codec.dart';
+
+void main() {
+  group('IntConverter Extension:', () {
+    test(
+        '5 as int will be converted to BigInt.from(5) when used with IntConverter Extension',
```

Comment:
Please consider using [BDD Naming conventio](https://matheus.ro/2017/09/24/unit-test-naming-convention/)n when creating your tests: **Given_Preconditions_When_StateUnderTest_Then_ExpectedBehavior **
```suggestion
        'Given any positive integer, it should convert to BigInt ',
```

Reply:
Interesting, why'd `dprint` block the pipeline?